zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/descriptors/IndexTemplateDescriptor.java

```diff
@@ -16,4 +16,14 @@ public interface IndexTemplateDescriptor extends IndexDescriptor {
   String getTemplateName();
 
   List<String> getComposedOf();
+
+  /**
+   * Maps to the create query parameter, if true the corresponding request cannot replace or update
+   * an existing component template, defaults to false
+   *
+   * @return The component template request is create only
+   */
+  default Boolean create() {
+    return false;
+  }
```

Comment:
I'm still unsure about this. If my understanding is correct, it's not up to the `IndexTemplate` to signal if it could be overwritten or not, but it's more about if we are creating or updating schema.

I think this doesn't belong here, and it should be passed as a parameter in the call stack to `PutIndexTemplate`

Reply:
Moved the create option to be passed in when creating and sending PutIndexTemplateRequest